src/renderer/api-decorator.js

```diff
@@ -429,6 +429,12 @@ limitations under the License.
         options.openfin = true;
         options.preload = 'preload' in options ? options.preload : winOpts.preload;
         options.uuid = winOpts.uuid;
+        if (options.backgroundColor === undefined &&
```

Comment:
@TomaszMa can you move this code here https://github.com/HadoukenIO/core/blob/develop/src/browser/api/window.js#L405

This is the unofficial place we've been using to handle the case where the child window should inherit an option from it's parent.

Reply:
If I could, I would move the code where you proposed, but check need to be done before we convert options to electron options (which is done few lines below my changes).